openreview/conference/templates/tabsConferenceDecisionsWebfield.js

```diff
@@ -8,55 +8,103 @@
 // Constants
 var CONFERENCE_ID = '';
 var BLIND_SUBMISSION_ID = '';
-var WITHDRAWN_INVITATION = '';
+var WITHDRAWN_SUBMISSION_ID = '';
+var DESK_REJECTED_SUBMISSION_ID = '';
 var DECISION_INVITATION_REGEX = '';
 var DECISION_HEADING_MAP = {};
-
+var PAGE_SIZE = 25;
 
 var HEADER = {};
 
+var paperDisplayOptions = {
+  pdfLink: true,
+  replyCount: true,
+  showContents: true
+};
+
+var sections = [];
+
 // Main is the entry point to the webfield code and runs everything
 function main() {
   Webfield.ui.setup('#group-container', CONFERENCE_ID);  // required
 
   renderConferenceHeader();
 
+  renderConferenceTabs();
+
   load().then(renderContent).then(Webfield.ui.done);
 }
 
 // Load makes all the API calls needed to get the data to render the page
 // It returns a jQuery deferred object: https://api.jquery.com/category/deferred-object/
 function load() {
-  var notesP = Webfield.getAll('/notes', { invitation: BLIND_SUBMISSION_ID, details: 'replyCount,original' });
 
-  var withdrawnNotesP = WITHDRAWN_INVITATION ? Webfield.getAll('/notes', { invitation: WITHDRAWN_INVITATION, noDetails: true }) : Promise.resolve({ notes: []});
+  var notesP = Webfield.api.getSubmissions(BLIND_SUBMISSION_ID, {
```

Comment:
it should be getAll instead of getSubmissions, we can not paginate the results from the server because we need to distribute them between the decisions tabs.